src/steps.tsx

```diff
@@ -382,31 +401,45 @@ const riskAssessment: Step[] = [
         trigger: 'howDialogueCanHelp'
       },
       {
-        label: 'What\'s "Dialogue"?',
+        label: "What's Dialogue?",
         value: false,
         trigger: 'explainDialogue'
       }
     ]
   },
   {
     id: 'howDialogueCanHelp',
-    message:
-      'Good for you! Remember you can always access Dialogue at https://app.dialogue.co to enquire about this or any other medical question you may have.',
+    message: `
+    Good for you! 
+    
+    Remember you can always access Dialogue at https://app.dialogue.co to enquire about this or any other medical question you may have.`,
     trigger: 'outro1'
   },
   {
     id: 'explainDialogue',
-    message:
-      "Dialogue is a virtual clinic for you and your family. Here's some of the thigs Dialogue can do for you: Chat with a nurse about any medical issue, skip the commute. Consult with a doctor via live video, skip the waiting room. Renew a prescription and get it delivered, skip a pharmacy queue. Get a referral to a specialist, fast. To learn more about Dialogue, visit https://dialogue.co",
+    message: `
+    Dialogue is a virtual clinic for you and your family. 
+      
+    Here's some of the thigs Dialogue can do for you: 
+    - Chat with a nurse about any medical issue, skip the commute. 
+    - Consult with a doctor via live video, skip the waiting room. 
+    - Renew a prescription and get it delivered, skip a pharmacy queue. 
+    - Get a referral to a specialist, fast. 
+      
+    To learn more about Dialogue, visit https://dialogue.co`,
     trigger: 'outro1'
   }
 ]
 
 const outro: Step[] = [
   {
     id: 'outro1',
-    message:
-      'Done with the questions. Your personal information package is ready. Check back using the link below, our medical team is updating the information as the situation develops. If you know other people in your situation, share the link with them.',
+    message: `
+    We're done with the questions! Your personal information package is ready.
+      
+    You can always check back using the link below, our medical team is updating the information as the situation develops. 
+      
+    If you know other people in your situation, share the link with them.`,
```

Comment:
this is encouraging them to share their summary's link, as opposed to the link to the bot. instead, can we add a 'share' button that links to the landing/bot?

Reply:
yeah, let's do that. I'll remove the line!

Reply:
also, we can embed an inline link to their summary here?